components/calendar/calendar.js

```diff
@@ -439,6 +439,7 @@ class Calendar extends LocalizeStaticMixin(RtlMixin(LitElement)) {
 				this._focusDateAddFocus();
 			} else if (prop === 'selectedValue') {
 				if (this.selectedValue) this._focusDate = getDateFromISODate(this.selectedValue);
+				else if (this._today.getMonth() === this._shownMonth) this._focusDate = new Date(this._today);
```

Comment:
Don't we also want to check the year? i.e. won't this select May 20 vs. May 1st if the shown year is 1999?